src/utils/AmountUtils.res

```diff
@@ -0,0 +1,39 @@
+let calculateBalance = (~transactions: array<TransactionType.t>) => {
+    Belt.Array.reduce(
+        transactions, 
+        0.0, 
+        (acc, value) => acc +. value.amount
+    )
+}
+
+let calculateIncome = (~transactions: array<TransactionType.t>) => {
+    Belt.Array.reduce(
+        transactions, 
+        0.0, 
+        (acc, value) => (value.amount >= 0. ? acc +. value.amount : acc +. 0.0)
+    )
+}
+
+let calculateExpense = (~transactions: array<TransactionType.t>) => {
+    Belt.Array.reduce(
+        transactions, 
+        0.0, 
+        (acc, value) => (value.amount < 0. ? acc +. value.amount : acc +. 0.0)
+    )
+}
+
+let check = (~amount) => {
+    amount >= 0. ? "" : "-"
+} 
+
+let isRounded = (~amount) => {
+    amount <= -10000000.0 || amount >= 10000000.0 ? true: false
+}
+
+let round = (~amount, ~isRoundedAmount) => {
+    isRoundedAmount ? Js.String.concat("K", Js.Float.toFixedWithPrecision((amount /. 10000000.0), ~digits=2)) : Belt.Float.toString(amount)
+}
+
+let refactor = (~amount) => {
```

Comment:
This function is removing the negative sign from the start of the amount. So that it could be added before the rupee sign i.e. this will convert $-400 to -$400.

Reply:
Have removed the function from here. Instead of using separate function for this, I just took the absolute value of the amount in the getShortenedAmount function which is used to shortened the amount and sign will be added at last using the getSign function.